content/blog/2020/2020-04-09-covid-19-part-2.md

```diff
@@ -0,0 +1,160 @@
+
+{{% sidework src="/blog_img/2020/2020-04-09-workflow-01.png" %}}
+
+Latitude and longitude are automatically selected. Below, you can define similar point properties, as in the scatter plot. The size of a point can intuitively represent the number of cases, so let's select the latest date in the *Size* dropdown. If you're feeling adventurous, try changing the map type to satellite or topological.
+
+{{% figure src="/blog_img/2020/2020-04-09-full-geomap.png" %}}
+
+Just like in Scatter Plot, you can scroll to zoom in and out, and click/drag to select data points. As opposed to using Scatter Plot, showing the points over a map of Europe makes it easier to select the countries we want to inspect.
+
+{{% figure src="/blog_img/2020/2020-04-09-eu-geomap.png" %}}
+
+## The chloro... chorophl... chloropet....... map with areas colored in.
+
+The map is nice and informative, and it's easy to distinguish larger epidemics from smaller ones. Still, those similarly-sized points on the map represent entire countries from big Russia to small Slovenia. We could, alternatively, represent countries by coloring their surface area. This is a job for Choropleth Map. Let's add it and connect it.
+
+{{% sidework src="/blog_img/2020/2020-04-09-workflow-02.png" %}}
+
+This widget is similar to Geo Map, but instead of displaying data as points, it aggregates them into regions (e.g., Country, State, Province), and displays those instead. Under *Attribute*, select the latest date and under *Agg.*, select *sum*. The colored regions now show the sum of confirmed COVID-19 cases for each country. 
+
+Try choosing different dates. Notice how China initially experiences rapid growth, followed by a period of stagnation, and how in the past week, the USA overtook all other countries' confirmed COVID-19 case count.
+
+{{% figure src="/blog_img/2020/2020-04-09-choropleth.png" %}}
+
+A warning that *7 points are not in any region* refers to some small islands and cruise ships included in the data.
+
+## If a picture paints a thousand words, an animation paints a million.
+
+When working with time-series data, the coolest insights arise from observing trends – how data changes depending on time. Let's animate our maps!
+
+We'll be using a special widget from Orange3-Timeseries: Time Slice, which lets you select data in a time window. It also features a playback function that slowly moves the time window, outputting corresponding data as it goes.
+
+But first, some widget kung-fu.
+
+### Kill Transpose: Volume 1
```

Comment:
I don't actually understand this title. Why 'Kill'?

Reply:
It was meant as a Kill Bill: Vol 1&2 reference because of the widget kung fu and the time slice.

Reply:
Oooooo. Missed this one. :) It could be just me, I was expecting you to describe how to remove Transpose from the workflow, as in 'kill' it.